services/softLayer_account.go

```diff
@@ -5,7 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-
+	"github.com/maximilien/softlayer-go/common"
```

Comment:
Usually I leave a space between imports that are from Golang, named and unnamed groups.